src/Sarif.Multitool/Rules/SARIF1002.UrisMustBeValid.cs

```diff
@@ -78,16 +82,30 @@ protected override void Analyze(VersionControlDetails versionControlDetails, str
 
         private void AnalyzeUri(Uri uri, string pointer)
         {
-            AnalyzeUri(uri?.OriginalString, pointer);
-        }
-
-        private void AnalyzeUri(string uri, string pointer)
-        {
-            if (uri != null)
+            string uriString = uri?.OriginalString;
+            if (uriString != null)
             {
-                if (!Uri.IsWellFormedUriString(uri, UriKind.RelativeOrAbsolute))
+                // UrisMustConformToRfc3986: URIs must conform to Rfc3986.
+                if (!Uri.IsWellFormedUriString(uriString, UriKind.RelativeOrAbsolute))
```

Comment:
After some thought I've decided that these logic-branch comments aren't necessary. The comments above the calls to `LogResult` suffice IMO. Note that my PR #1934 doesn't include these branch comments.